test-suite/output/Cases.out

```diff
@@ -232,3 +232,12 @@ match x with
 | @D' _ _ _ _ n _ p _ => (n, p)
 end
      : J' bool (true, true) -> nat * nat
+File "./test-suite/output/Cases.v", line 313, characters 3-4:
```

Comment:
The location doesn't seem to be correct, the CI prints stdin instead.

Reply:
Thanks, fixed. Btw, I did not find yet how to test a single file of the test-suite with dune.